### ISSUE TITLE
Pre trained models

### ISSUE THREAD

**Issue by joanroig** [aliutkus/spe#12]
I would like to ask for the pre trained models of the pop piano experiment used in the paper, could you please provide them?
Thanks

**Comment by cifkao** [comment]
@slSeanWU

**Comment by slSeanWU** [comment]
To resolve the OOM issue, you may either reduce `n_layer` or `max_len` in the config.

For another error arising during loss computation, you may try changing `line 63, 133 in train.py` from
```python
batch_dec_tgt = batch_samples['dec_target'].cuda(gpuid)
```
to
```python
batch_dec_tgt = batch_samples['dec_target'].long().cuda(gpuid)
```

Let me know whether this works!

**Comment by joanroig** [comment]
@slSeanWU I confirm that applying the fix and reducing the `n_layer` and `max_len` lets me start training the original dataset.
An extra fix I needed to apply in the same file:
batch_dec_tgt = batch_samples['dec_target'].long().cuda(gpuid).permute(1, 0)

**Comment by slSeanWU** [comment]
Hi,

Just one thing, the `permute(1, 0)` should actually be removed from the validate() function, otherwise we would get wrong validation loss. I've pushed the fix.

And of course, you may try fine-tuning, though I'm not sure how well it'll work.

**Comment by joanroig** [comment]
@slSeanWU I'm not experienced enough to successfully setup a finetune process, so for now I'm very happy to train from scratch and try some outputs from your weights, thanks :)
